chore: add structured package data for `math/base/special/acoth`

nakul-krishnakumar opened the pull request:
---
type: pre_commit_static_analysis_report
description: Results of running static analysis checks when committing changes. report:
  - task: lint_filenames status: passed
  - task: lint_editorconfig status: passed
  - task: lint_markdown status: na
  - task: lint_package_json status: passed
  - task: lint_repl_help status: na
  - task: lint_javascript_src status: na
  - task: lint_javascript_cli status: na
  - task: lint_javascript_examples status: na
  - task: lint_javascript_tests status: na
  - task: lint_javascript_benchmarks status: na
  - task: lint_python status: na
  - task: lint_r status: na
  - task: lint_c_src status: na
  - task: lint_c_examples status: na
  - task: lint_c_benchmarks status: na
  - task: lint_c_tests_fixtures status: na
  - task: lint_shell status: na
  - task: lint_typescript_declarations status: na
  - task: lint_typescript_tests status: na
  - task: lint_license_headers status: passed ---

Resolves a part of #7924 .

## Description

> What is the purpose of this pull request?

This pull request:

-   Adds structured package data for `math/base/special/acoth`.

## Related Issues

> Does this pull request have any related issues?

This pull request:

-   resolves a part of #7924 

## Questions

> Any questions for reviewers of this pull request?

No.

## Other

> Any other information relevant to this pull request? This may include screenshots, references, and/or implementation notes.

No.

## Checklist

> Please ensure the following tasks are completed before submitting this pull request.

-   [x] Read, understood, and followed the [contributing guidelines][contributing].

* * *

@stdlib-js/reviewers

[contributing]: https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md

nakul-krishnakumar committed:
chore: add structured package data for `math/base/special/acoth`

---
type: pre_commit_static_analysis_report
description: Results of running static analysis checks when committing changes.
report:
  - task: lint_filenames
    status: passed
  - task: lint_editorconfig
    status: passed
  - task: lint_markdown
    status: na
  - task: lint_package_json
    status: passed
  - task: lint_repl_help
    status: na
  - task: lint_javascript_src
    status: na
  - task: lint_javascript_cli
    status: na
  - task: lint_javascript_examples
    status: na
  - task: lint_javascript_tests
    status: na
  - task: lint_javascript_benchmarks
    status: na
  - task: lint_python
    status: na
  - task: lint_r
    status: na
  - task: lint_c_src
    status: na
  - task: lint_c_examples
    status: na
  - task: lint_c_benchmarks
    status: na
  - task: lint_c_tests_fixtures
    status: na
  - task: lint_shell
    status: na
  - task: lint_typescript_declarations
    status: na
  - task: lint_typescript_tests
    status: na
  - task: lint_license_headers
    status: passed
---

gunjjoshi commented on lib/node_modules/@stdlib/math/base/special/acoth/package.json:
```diff
@@ -70,5 +70,92 @@
+            "prng": "random/base/uniform",
+            "parameters": [
+              -10,
+              10
```
@nakul-krishnakumar We might need to change this, as it contains the interval `(-1, 1)`, where the function returns `NaN`.

gunjjoshi commented on lib/node_modules/@stdlib/math/base/special/acoth/package.json:
```diff
@@ -70,5 +70,92 @@
+            64,
+            27,
+            0,
+            0.1,
```
These two values should be removed, as they lie in `(-1, 1)`, where the function is not defined.

nakul-krishnakumar replied:
Got it, thanks for the review!

nakul-krishnakumar committed:
refactor: apply suggestions from code review

---
type: pre_commit_static_analysis_report
description: Results of running static analysis checks when committing changes.
report:
  - task: lint_filenames
    status: passed
  - task: lint_editorconfig
    status: passed
  - task: lint_markdown
    status: na
  - task: lint_package_json
    status: passed
  - task: lint_repl_help
    status: na
  - task: lint_javascript_src
    status: na
  - task: lint_javascript_cli
    status: na
  - task: lint_javascript_examples
    status: na
  - task: lint_javascript_tests
    status: na
  - task: lint_javascript_benchmarks
    status: na
  - task: lint_python
    status: na
  - task: lint_r
    status: na
  - task: lint_c_src
    status: na
  - task: lint_c_examples
    status: na
  - task: lint_c_benchmarks
    status: na
  - task: lint_c_tests_fixtures
    status: na
  - task: lint_shell
    status: na
  - task: lint_typescript_declarations
    status: na
  - task: lint_typescript_tests
    status: na
  - task: lint_license_headers
    status: passed
---

kgryte commented on lib/node_modules/@stdlib/math/base/special/acoth/package.json:
```diff
@@ -70,5 +70,90 @@
+              "max": "-1"
+            },
+            {
+              "min": "1",
```
```suggestion
              "max": -1
            },
            {
              "min": 1,
```

These literal values do not need to be wrapped in strings. The reason `infinity` is wrapped in a string is that it is not valid JSON as a number.

kgryte committed:
fix: unwrap number literals

Signed-off-by: Athan <kgryte@gmail.com>

kgryte commented on lib/node_modules/@stdlib/math/base/special/acoth/package.json:
```diff
@@ -70,5 +70,90 @@
+        "angle"
+      ],
+      "extra_keywords": [
+        "math.acot"
```
```suggestion
        "math.atanh"
```

`acot` doesn't exist on the built-in JavaScript `math` object. Ref: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math

kgryte committed:
docs: update keywords

Signed-off-by: Athan <kgryte@gmail.com>

kgryte reviewed (dismissed):
LGTM.

kgryte commented on lib/node_modules/@stdlib/math/base/special/acoth/package.json:
```diff
@@ -70,5 +70,90 @@
+            -188,
+            19.11,
+            -200,
+            21.15
```
```suggestion
            21.15,
            -1.2,
            1.7
```

kgryte committed:
chore: add values to ensure `20` members

Signed-off-by: Athan <kgryte@gmail.com>
